src/app/app.constants.ts

```diff
@@ -205,6 +205,8 @@ const appHeaderTitle = {name: 'Manage cases', url: '/'};
 // Making the base URL for case details a constant.
 const caseDetailsUrl: string = '/cases/case-details/';
 
+// Create Case button text
+const buttonText: string = 'Start now';
```

Comment:
Sorry to be picky, but it might be better to have a more descriptive name for this. Such as `startButtonText`... `buttonText` is quite vague.

Reply:
Done